packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -690,6 +690,81 @@ void main() {
     await tester.tap(find.text('Menu Button'));
   });
 
+  testWidgets('PopupMenuItem child height is a minium, child is vertically centered', (WidgetTester tester) async {
+    // This is a regression test for https://github.com/flutter/flutter/issues/39508.
+
+    final Key popupMenuButtonKey = UniqueKey();
+    final Type menuItemType = const PopupMenuItem<String>(child: Text('item')).runtimeType;
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Scaffold(
+          body: Center(
+            child: PopupMenuButton<String>(
+              key: popupMenuButtonKey,
+              child: const Text('button'),
+              onSelected: (String result) { },
+              itemBuilder: (BuildContext context) {
+                return <PopupMenuEntry<String>>[
+                  const PopupMenuItem<String>( // Item's height = 48 (height == minHeight is 48).
+                    child: Text('Item 0'),
+                    value: '0',
+                  ),
+                  const PopupMenuItem<String>( // Item's height = 50 (height == minHeight is 50).
```

Comment:
I feel like the comments beyond the first item are not as helpful (first one is helpful because it points out the default min height), since `PopupMenuItem.height` documentation clarifies that it sets the minimum height.

Reply:
Or maybe define what `SizedBox` wants to be, but what ends up happening here and in the example below